### DISCUSSION TITLE
Prawn::Errors::IncompatibleStringEncoding: Your document includes text that's not compatible with the Windows-1252 character set. If you need full UTF-8 support, use TTF fonts instead of PDF's built-in fonts

### DISCUSSION THREAD

**Discussion by aamir-nazir** [prawnpdf/prawn#1212] [General]
I ran into this issue. Program was trying to render a text which contain non-Ascii character in the end i.e 65532 and generating this error. 

I was wondering if Prawn has any configuration which sanitise text and remove illegal/not supported chars at a common point?

**Comment by pointlessone** [comment]
No, Prawn doesn't provide any text transformation/filtering tools. You'll have to do it on your own.

The reason is that there are a few potential methods with different outcomes and none of them is obviously correct. You might want to remove those characters, you might want to instead use an external font that can render those characters. It's all up to you.
